src/fmu/sumo/uploader/scripts/sumo_upload.py

```diff
@@ -23,21 +23,55 @@
 # This documentation is for sumo_uploader as an ERT workflow
 DESCRIPTION = """SUMO_UPLOAD will upload files to Sumo. The typical use case is as add-on to
 post-processing workflows which aggregate data across an ensemble and stores the
-results outside the realization folders.
+results outside the realization folders. 
+
+SUMO_UPLOAD depends on ``fmu-dataio`` and ``WF_CREATE_CASE_METADATA``. 
```

Comment:
Maybe "SUMO_UPLOAD depends on the current case being registered in Sumo (done through the `WF_CREATE_CASE_METADATA` workflow) and on data being exported with `fmu-dataio`.